supervisor/shared/board_busses.c

```diff
@@ -31,17 +31,20 @@
 #include "shared-bindings/microcontroller/Pin.h"
 #include "supervisor/shared/translate.h"
 #include "mpconfigboard.h"
-#include "samd/pins.h"
 #include "py/runtime.h"
 
+#ifdef CIRCUITPY_DISPLAYIO
+#include "shared-module/displayio/__init__.h"
+#endif
+
 #define BOARD_I2C (defined(DEFAULT_I2C_BUS_SDA) && defined(DEFAULT_I2C_BUS_SCL))
 #define BOARD_SPI (defined(DEFAULT_SPI_BUS_SCK) && defined(DEFAULT_SPI_BUS_MISO) && defined(DEFAULT_SPI_BUS_MOSI))
 #define BOARD_UART (defined(DEFAULT_UART_BUS_RX) && defined(DEFAULT_UART_BUS_TX))
 
 #if BOARD_I2C
 STATIC mp_obj_t i2c_singleton = NULL;
 
-STATIC mp_obj_t board_i2c(void) {
+mp_obj_t board_i2c(void) {
 
     if (i2c_singleton == NULL) {
         busio_i2c_obj_t *self = m_new_obj(busio_i2c_obj_t);
```

Comment:
Make this long-lived?

Reply:
I'd rather not in this PR. It already has a lot of extra stuff in it.

Reply:
np